src/components/List/List.style.ts

```diff
@@ -30,9 +30,10 @@ export const listStyle =
       margin-bottom: 0;
       border-radius: ${isSearchable ? 'initial' : theme.globals.spacing.get('3')};
       width: ${width ? rem(width) : '100%'};
-      height: ${height ? rem(height) : '100%'};
+      height: ${height ? rem(height) : 'auto'};
       overflow: auto;
       overflow-x: hidden;
+      background: #fff;
```

Comment:
you can use `theme.globals.colors.white` for consistency

Reply:
solved at [`3b16bc4` (#655)](https://github.com/Orfium/orfium-ictinus/pull/655/commits/448650c8b6b57c71be2e74c89a0e93a88c23f2a8)